repl/src/main/scala-2.13/org/apache/spark/repl/SparkILoop.scala

```diff
@@ -17,6 +17,8 @@
 
 package org.apache.spark.repl
 
+import org.apache.spark.repl.InactivityTimeout.{startInactivityTimer, stopInactivityTimer}
```

Comment:
Please import just `InactivityTimeout` and use `InactivityTimeout.startInactivityTimer` and `InactivityTimeout.stopInactivityTimer` where it is needed.